### PR TITLE
Added Two things that happened in the Rust China community this week

### PR DESCRIPTION
**Pull request by ZhangHanDong**
- [[ZH] Rust Chinese Magazine Issue 10 has been released!](https://rustmagazine.github.io/rust_magazine_2021/chapter_10/toc.html)
- [[ZH] The first draft of the Rust Secure Coding Specification in Chinese has been released. Everyone is welcome to contribute](https://rust-coding-guidelines.github.io/rust-coding-guidelines-zh/)
- [[ZH] Trojan Source｜ Hiding invisible vulnerabilities in Rust code](https://zhuanlan.zhihu.com/p/428305373)

### CODE REVIEW

**Comment by nellshamrell** [comment]
@MATRIXKOO and @Folyd do you mind helping me review these two articles?

**Review by PsiACE — APPROVED** [review]
LGTM, cc @nellshamrell

File: draft/2021-11-03-this-week-in-rust.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 ### Observations/Thoughts
 
+[[ZH] Rust Chinese Magazine Issue 10 has been released!](https://rustmagazine.github.io/rust_magazine_2021/chapter_10/toc.html)
+[[ZH] The first draft of the Rust Secure Coding Specification in Chinese has been released. Everyone is welcome to contribute](https://rust-coding-guidelines.github.io/rust-coding-guidelines-zh/)
+
+!](https://rustmagazine.github.io/rust_magazine_2021/chapter_10/toc.html)
```
Comment (PsiACE) [inline draft/2021-11-03-this-week-in-rust.md:28]: pls fix format and display, cc @ZhangHanDong

**Review by PsiACE — CHANGES_REQUESTED** [review]

**Comment by ZhangHanDong** [comment]
A new article has been added, a total of three articles

**Comment by MATRIXKOO** [comment]
Greate guidelines. 
And [\[ZH\] Trojan Source｜ Hiding invisible vulnerabilities in Rust code](https://zhuanlan.zhihu.com/p/428305373) is an interpretation for Chinese readers.

**Review by MATRIXKOO — COMMENTED** [review]
[\[ZH\] Trojan Source｜ Hiding invisible vulnerabilities in Rust code](https://zhuanlan.zhihu.com/p/428305373) viewed.

**Review by Folyd — APPROVED** [review]
LGTM.

**Comment by nellshamrell** [comment]
Thank you so much, @MATRIXKOO and @Folyd!

**Pull request merged**